### DISCUSSION TITLE
Renseignements concernant les IP

### DISCUSSION THREAD

**Discussion by victorgyt** [dbarzin/mercator#658] [Q&A]
Bonjour, 

Depuis quelque jours nous recherchons à renseigner les adresses IP de nos cartes iLO ou de tout autre carte type d'accès à distance (KVM, etc). 
Pouvez vous nous dire où nous pourrions renseigner ces informations ? 

En vous remerciant, 

Bien cordialement 
V.G

**Comment by dbarzin** [comment]
Vous pouvez ajouter cette information dans les équipements de sécurité de l'infrastructure logique.
Mais il faut que j'ajoute l'adresse IP dans cette table. 
Pour les KVM et autres ce sont des équipements de sécurité physique.

**Comment by victorgyt** [comment]
Bonjour, 

Merci pour votre retour rapide. 

Où pourrais-je glisser les informations type "PONT FH" ? 

Merci ! 

Cordialement

**Comment by dbarzin** [comment]
Dans la table "équipement de sécurité logique", il existe un champ "Caractéristiques techniques" qui peut contenir cette information.
